web/mass_class/views.py

```
@@ -0,0 +1,57 @@
+# web/mass_class/views.py
+from django.contrib.auth.decorators import login_required
+from django.http import JsonResponse
+from django.shortcuts import get_object_or_404, redirect, render
+from django.urls import reverse
+from django.utils import timezone
+
+from ..models import Course, MassClassStream, Session
+
+
+@login_required
+def teacher_broadcast_view(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `teacher_broadcast_view`

(ANN201)